frontend/app/scripts/modules/topology/controllers/topologyCtrl.js

```diff
@@ -114,7 +114,7 @@ angular.module('serviceCenter.topology', [])
         }
 
         function createTopology() {
-            var microServices = new vis.DataSet($scope.microServices);
+            var microServices = new vis.DataSet($scope.microServices.slice(0,100));
```

Comment:
And How do User view after 100 microservices?

Reply:
@xiaoyimei If the total number of Microservices in the complete env increases more than 100 then we need to force User to select only 1 APP-ID and then the topology will show the microservices only for that particular APP.

Reply:
sorry asif, I didn’t understand your meaning correctly before, but I understand it now.